packages/core/src/components/Modal/Modal.styled.tsx

```diff
@@ -58,7 +60,7 @@ export const InnerContainerStyled = styled.div<InnerContainerProps>`
     }
 
     @media (min-width: 768px) {
-        overflow: hidden;
+        ${({ overflowVisible }) => !overflowVisible && `overflow: hidden;`};
```

Comment:
You can write like this overflow: ${({ overflowVisible }) => !overflowVisible && 'hidden'}; for better readability.